src/cosmology/compat/camb/__init__.py

```diff
@@ -1,24 +1,110 @@
-"""The Cosmology API compatability library for :mod:`camb`.
+"""Cosmology API compatibility layer for CAMB."""
 
-This library provides wrappers for CAMB cosmology objects to be compatible
-with the Cosmology API. The available wrappers are:
+from __future__ import annotations
 
-- :class:`.CAMBCosmology`: the Cosmology API wrapper for
-  :mod:`astropy.camb.Cosmology`.
+from dataclasses import dataclass
 
+import numpy as np
 
-There are the following required objects for a Cosmology-API compatible library:
+TYPE_CHECKING = False
+if TYPE_CHECKING:
+    from typing import TypeAlias
 
-- constants: a module of constants. See
-  :mod:`cosmology.compat.camb.constants` for details.
-"""
+    from numpy.typing import NDArray
 
-from . import constants
-from .core import CAMBCosmology
+    from camb import CAMBdata, CAMBparams  # noqa: TC001
 
-__all__ = [
-    # Cosmology API
-    "constants",
-    # Wrappers
-    "CAMBCosmology",
-]
+    Array: TypeAlias = NDArray[np.float64]
+
+
+@dataclass
+class Cosmology:
+    """Cosmology API wrapper for CAMB *pars* and *results*."""
+
+    pars: CAMBparams
+    results: CAMBdata
+
+    @property
+    def h(self) -> Array:
+        """Little h."""
+        return np.array(self.pars.h)
```

Comment:
I'm a huge fan of doctests being an important source of unit tests!
Users and devs get good examples, devs get tests rolled in the same place as the code. Win win.
All that to say, docstring tests here and elsewhere to hit the codecov?

Reply:
In the end I went with comparing against astropy.cosmology since that gives a bit of external verification that the code doesn't just call some methods from CAMB, but also that their meaning is correct.